### ISSUE TITLE
SES6 Deployment: add msgr2 enablement to "Upgrade Ceph OSDs" section

### ISSUE THREAD

**Issue by smithfarm** [SUSE/doc-ses#314]
**Link To Section**

https://documentation.suse.com/ses/6/single-html/ses-deployment/#upgrade-main-osd

**Summary**

Now that msgr2 is fully supported in SES6 [1], we need to instruct users to enable it at the appropriate point in the SES5.5 -> 6 upgrade procedure, using the instructions at [2].

[1] https://github.com/SUSE/doc-ses/issues/310
[2] https://ceph.io/releases/v14-2-0-nautilus-released/#instructions

**What Exact Changes To Make To the Upgrade Procedure**

In the section "Upgrade Ceph OSDs" of the SES6 Deployment Guide, at step 9 all of the OSD nodes will have been upgraded and the user will be checking the cluster health. There, they will notice (and this is correctly reflected in the documentation) a health warning message:

      health: HEALTH_WARN
          3 monitors have not enabled msgr2

To address this and another issue, let us add new steps 10 and 11 (after step 9, but before the existing step 10, which will be renumbered as a result of inserting these new step):

10. Once the last OSD node has been upgraded, issue the following command

        ceph osd require-osd-release nautilus

    This disallows pre-SES6/Nautilus OSDs and enables all new SES6/Nautilus-only OSD functionality.

11. Enable the new v2 network protocol by issuing the following command:

        ceph mon enable-msgr2

    This will instruct all monitors that bind to the old default port 6789 for the legacy `v1` Messenger protocol to also bind to the new 3300 `v2` protocol port. To see if all monitors have been updated, do:

        ceph mon dump

    and verify that each monitor has both a `v2:` and `v1:` address listed.

**Comment by smithfarm** [comment]
@jan--f @tserong Does this look right to you?

**Comment by tserong** [comment]
LGTM, with one small kink -- DeepSea currently sets `ms_bind_msgr2 = false` in /etc/ceph/ceph.conf. I haven't tested, but I assume that overrides `ceph mon enable-msgr2`?

**Comment by smithfarm** [comment]
> LGTM, with one small kink -- DeepSea currently sets `ms_bind_msgr2 = false` in /etc/ceph/ceph.conf. I haven't tested, but I assume that overrides `ceph mon enable-msgr2`?

My understanding is no, it does not override. The two are orthogonal, and `ms_bind_msgr2 = false` doesn't do what it's expected to do (it's ignored by MONs, for example).

Also, DeepSea adding that line is currently a noop because SES6 has this commit: https://github.com/SUSE/ceph/commit/e2c4e78b459e2a9c32f50316b9e273a341290ade

Once these doc changes go through, though, we should revert both DeepSea and Ceph so they no longer set `ms_bind_msgr2 = false`.
